mmv1/products/alloydb/Cluster.yaml

```diff
@@ -478,7 +479,6 @@ properties:
       - :PRIMARY
       - :SECONDARY
     default_value: :PRIMARY
-    immutable: true
```

Comment:
Note: To perform secondary cluster promotion, a user would modify the `cluster_type` field from `SECONDARY` to `PRIMARY`. The promotion logic is implemented by modifying the update steps in `pre_update`. So this field needs to mutable.